assets/js/modules/ads/index.js

```diff
@@ -141,14 +141,19 @@ export const ADS_NOTIFICATIONS = {
 			// isWooCommerceActivated, isGoogleForWooCommerceActivated and isGoogleForWooCommerceLinked are all relying
 			// on the data being resolved in getModuleData() selector.
 			await resolveSelect( MODULES_ADS ).getModuleData();
+			await resolveSelect( CORE_MODULES ).isModuleConnected( 'ads' );
 
 			const {
 				isWooCommerceActivated,
 				isGoogleForWooCommerceActivated,
 				hasGoogleForWooCommerceAdsAccount,
 			} = select( MODULES_ADS );
 
+			const isModuleConnected =
+				select( CORE_MODULES ).isModuleConnected( 'ads' );
```

Comment:
You don't need to call the `isModuleConnected` selector twice, its value is returned from the `resovleSelect` call as well.

```js
const [ _, isModuleConnected ] = await Promise.all( [
    resolveSelect( MODULES_ADS ).getModuleData(),
    resolveSelect( CORE_MODULES ).isModuleConnected( 'ads' ),
] );
```